### ISSUE TITLE
question about the position setting、texure、and the glTF supporting

### ISSUE THREAD

**Issue by huangsn10** [opensourceBIM/BIMsurfer#55]
I have 3 questions: 
1\about  .glTF ,  can the latest bimserver version import .glTF file?  I have tried but just found that it can export .glTF file

2\about  Texure , can the V3 Bimsurfer  Support adding image file（like jpg、png etc） ?or are there any api can do that?

3\about  position， now can not change the element's position after loaded, but will it be support in the future update?

**Comment by rubendel** [comment]
Please use a more descriptive issue title.

1. BIMserver cannot _import_ .glTF, but since you are on the BIMsurfer issue tracker, I assume you mean BIMsurfer. BIMsurfer 3 cannot import .glTF files, PRs are welcome
2. No, I have yet to see the first IFC file with textures, so I am guessing no authoring tools are exporting them yet, let's wait for that first I'd say, but again PRs are welcome.
3. This I already answered via email, no, the nature of this viewer with it's focus on performance does not allow for changing the positions of objects, also not in a future update. I can imagine this to be implemented for a small set of objects without losing performance, but it would be quite some work.

**Comment by huangsn10** [comment]
sorry for that title and I have changed it already.

 thanks a lot for replying ，as you said in mail, I repeat here  to let others with the same question  know the answer,  so pls dont care about that, sure I got the answer already  and you have done a great job.   And I will be happy to see that some day bimsurfer can suport position changing together with color/hide/show editing even for a small set of object, it will be pretty cool then.

and sorry again for my ignorance about the word PRs, what is it meaning?
